### PR TITLE
Stats

### PR DESCRIPTION
**Pull request by AkashAryal**
### Summary
This pull request adds a statistics component to the admin dashboard. Currently has two tables. One table shows how many reviews a class has (sorted in descending order). Another table shows how many classes are in each department (also sorted in descending order). Total num of reviews in db is a listed statistic.

This is the first part of a bigger project to track how much progress we are making in terms of students using our site. In the future charts will be added to track reviews over time.

### Test Plan
This feature was tested before hand locally. Can also test in the staging site by comparing value listed in table to expected values.

Nothing in this pull request should cause problems to existing parts of the site.

### CODE REVIEW

File: imports/ui/Accordian.jsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+                  {this.props.title}
+                </a>
+              </h4>
+            </div>
+            <div id={collapseOneId} className="panel-collapse collapse in" role="tabpanel" aria-labelledby="headingOne">
+              <div className="panel-body">
+              <table className="table table-striped">
+                <thead>
+                  <tr>
+                    <th scope="col">{this.props.col1}</th>
```
Comment (SamChou19815) [inline imports/ui/Accordian.jsx]: See the files changed tab for all linter warnings. Let's avoid introducing more linter warnings.

File: imports/ui/Statistics.jsx [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  reviews: PropTypes.array.isRequired*/
+  //totalCs: PropTypes.number.isRequired
+//};
+
+/*export default withTracker(props => {
+  const totalCS = Reviews.find({classSub: "cs"}).count();
+  return {
+    totalCS
+  };
+}) (Statistics);*/
```
Comment (SamChou19815) [inline imports/ui/Statistics.jsx]: Commented code is bad.

**Comment by SamChou19815** [comment]
@AkashAryal Your git might not be properly configured. You should do
```bash
git config --global user.email [your email used to register GitHub]
```
You might want to do some git commit rewrite to fix this PR's commit, otherwise the CLA check might fail (since it only has your ID AkashAryal on record).

File: imports/ui/Accordian.jsx [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+import PropTypes from 'prop-types';
+
+/*
+  An accordian for the Statistics component.
+*/
+export default class Accordian extends Component{
+
+  getRandNum(){
+    return Math.floor((Math.random() * 1000000) + 0);
+  }
```
Comment (SamChou19815) [inline imports/ui/Accordian.jsx]: The indentation is inconsistent with the rest. We might want to enforce consistent indentation in the linter.

File: imports/ui/Accordian.jsx (hunk 1 above)
Comment (julian-londono) [reply imports/ui/Accordian.jsx]: Fixed, will make sure Akash (@AkashAryal) knows to check out linter for next time.

File: imports/ui/Accordian.jsx (hunk 3 above)
Comment (julian-londono) [reply imports/ui/Accordian.jsx]: Will look into adding indentation to linter

**Pull request merged**